README.md

```diff
@@ -1385,12 +1384,12 @@ The `cleanup` workflow can be manually run to delete all resources created by th
 
 ## Next Steps
 
-In this quickstart, you've deployed polyglot applications to Azure Spring Cloud using Azure CLI.
+In this quickstart, you've deployed polyglot applications to Azure Spring Apps using Azure CLI.
 You also configured VMware Tanzu components in the enterprise tier. To learn more about
-Azure Spring Cloud or VMware Tanzu components, go to:
+Azure Spring Apps or VMware Tanzu components, go to:
 
-* [Azure Spring Cloud](https://azure.microsoft.com/en-us/services/spring-cloud/)
-* [Azure Spring Cloud docs](https://docs.microsoft.com/en-us/azure/spring-cloud/quickstart-provision-service-instance-enterprise?tabs=azure-portal)
+* [Azure Spring Apps](https://azure.microsoft.com/en-us/services/spring-cloud/)
+* [Azure Spring Apps docs](https://docs.microsoft.com/en-us/azure/spring-cloud/quickstart-provision-service-instance-enterprise?tabs=azure-portal)
```

Comment:
Assuming microsoft is going to redirect all this stuff, just a note for later that we may need to handle it